testcases/OpTestLPM.py

```diff
@@ -138,7 +138,7 @@ def check_pkg_installation(self):
         pkg_found = True
         pkg_notfound = []
         self.oslevel = self.cv_HOST.host_get_OS_Level()
-        lpm_pkg_list = ["src", "rsct.core", "rsct.core.utils", "rsct.basic", "rsct.opt.storagerm", "DynamicRM", "stress-ng"]
+        lpm_pkg_list = ["src", "rsct.core", "rsct.core.utils", "rsct.basic", "rsct.opt.storagerm", "DynamicRM", "stress-ng", "sos", "supportutils"]
         for pkg in lpm_pkg_list:
             pkg_status = self.cv_HOST.host_check_pkg_installed(self.oslevel, pkg)
```

Comment:
pkg_status will fail if sos package is not in SUSE OS.. and support in RHEL OS.. so we need to take care explicitly